### PR TITLE
Fix broken links in repo

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to @22375.

### CODE REVIEW

File: docs/architecture/microservices/architect-microservice-container-applications/maintain-microservice-apis.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ ms.date: 01/13/2021
 
 A microservice API is a contract between the service and its clients. You'll be able to evolve a microservice independently only if you do not break its API contract, which is why the contract is so important. If you change the contract, it will impact your client applications or your API Gateway.
 
-The nature of the API definition depends on which protocol you're using. For instance, if you're using messaging (like [AMQP](http://www.amqp.org/)), the API consists of the message types. If you're using HTTP and RESTful services, the API consists of the URLs and the request and response JSON formats.
+The nature of the API definition depends on which protocol you're using. For instance, if you're using messaging (like [AMQP](https://www.amqp.org)), the API consists of the message types. If you're using HTTP and RESTful services, the API consists of the URLs and the request and response JSON formats.
```
Comment (tdykstra) [inline docs/architecture/microservices/architect-microservice-container-applications/maintain-microservice-apis.md:10]: This link doesn't work with https

**Review by tdykstra — APPROVED** [review]
LGTM

File: docs/architecture/microservices/architect-microservice-container-applications/maintain-microservice-apis.md (hunk 1 above)
Comment (gewarren) [reply docs/architecture/microservices/architect-microservice-container-applications/maintain-microservice-apis.md:10]: Does it work for you without? I couldn't get it to work either way.

Comment (tdykstra) [reply docs/architecture/microservices/architect-microservice-container-applications/maintain-microservice-apis.md]: No, just tried http and got the same result.

File: docs/fsharp/tutorials/type-providers/creating-a-type-provider.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ The type provider mechanism in F# is a significant part of its support for infor
 
 The F# ecosystem contains a range of type providers for commonly used Internet and enterprise data services. For example:
 
-- [FSharp.Data](https://fsharp.github.io/FSharp.Data/) includes type providers for JSON, XML, CSV and HTML document formats.
+- FSharp.Data includes type providers for JSON, XML, CSV and HTML document formats.
```
Comment (Youssef1313) [inline docs/fsharp/tutorials/type-providers/creating-a-type-provider.md:13]: These links may be broken unintentionally.

cc: @cartermp

**Review by karelz — APPROVED** [review]
HttpWebRequest file change LGTM

**Pull request merged**